core/utils/poly-look/src/react-components/tabs/tabs.jsx

```diff
@@ -13,6 +13,7 @@ const Tabs = ({ children, swipe = true }) => {
     ev.preventDefault();
     swiperRef.current.swiper.slideTo(index);
     setActiveTabId(newActiveTabId);
+    onTabChange(newActiveTabId);
```

Comment:
OK, the default value of this function is an empty function; however, it's called with an argument. Maybe could you reflect that in the default somehow?

Reply:
Might just put an if statement and execute the function if it exists rather than putting a default function